py-polars/polars/dataframe/groupby.py

```diff
@@ -676,6 +676,35 @@ def n_unique(self) -> DF:
         """
         return self.agg(F.all().n_unique())
 
+    def approx_unique(self) -> DF:
```

Comment:
We shouldn't expose this on the `GroupBy`. I don't expect this to work with all dtypes, and we don't do this with most expressions.

Reply:
done